modules/dynamic-plugins/rhdh-supported-plugins.adoc

```diff
@@ -4,86 +4,206 @@
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-quay
+
+
+|Quay  |https://npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-quay/v/1.5.0[@janus-idp/backstage-scaffolder-backend-module-quay] |1.5.0 
+|./dynamic-plugins/dist/janus-idp-backstage-scaffolder-backend-module-quay-dynamic
+
+
+|RBAC  |https://npmjs.com/package/@janus-idp/backstage-plugin-rbac/v/1.24.1[@janus-idp/backstage-plugin-rbac] |1.24.1 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-rbac
+
+
+|Regex  |https://npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-regex/v/1.5.0[@janus-idp/backstage-scaffolder-backend-module-regex] |1.5.0 
+|./dynamic-plugins/dist/janus-idp-backstage-scaffolder-backend-module-regex-dynamic
+
+
+|TechDocs  |https://npmjs.com/package/@backstage/plugin-techdocs/v/1.10.5[@backstage/plugin-techdocs] |1.10.5 
+|./dynamic-plugins/dist/backstage-plugin-techdocs
+
+
+|TechDocs  |https://npmjs.com/package/@backstage/plugin-techdocs-backend/v/1.10.5[@backstage/plugin-techdocs-backend] |1.10.5 
+|./dynamic-plugins/dist/backstage-plugin-techdocs-backend-dynamic
+
+
+|Tekton  |https://npmjs.com/package/@janus-idp/backstage-plugin-tekton/v/3.8.0[@janus-idp/backstage-plugin-tekton] |3.8.0 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-tekton
+
+
+|Topology  |https://npmjs.com/package/@janus-idp/backstage-plugin-topology/v/1.22.0[@janus-idp/backstage-plugin-topology] |1.22.0 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-topology
+
+
+|===
+[NOTE]
+====
+* To configure Keycloak, see xref:rhdh-keycloak_{context}[Installation and configuration of Keycloak].
+
+* To configure Techdocs, see http://backstage.io/docs/features/techdocs/configuration[reference documentation]. After experimenting with basic setup, use CI/CD to generate docs and an external cloud storage when deploying TechDocs for production use-case.
+See also this https://backstage.io/docs/features/techdocs/how-to-guides#how-to-migrate-from-techdocs-basic-to-recommended-deployment-approach[recommended deployment approach].
+====
+
+== Technology Preview plugins
+
+=== Red Hat plugins
+
 [IMPORTANT]
 ====
-Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend using them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+Red Hat includes a select number of Technology Preview plugins, available for customers to configure and enable. These %%COUNT_2%% included plugins and are provided with support scoped per Technical Preview terms, might not be functionally complete, and Red Hat does not recommend using them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```

Comment:
@nickboldt Should these _%%COUNT_2%%_ placeholders be populated in Github previews (they're not)...also just wondering if they will render correctly in Pantheon?

Reply:
nice catch but no, they're only in the template... once I finish wiring up the code